vic/drivers/image/src/vic_init.c

```diff
@@ -57,6 +57,7 @@ vic_init(void)
     size_t                     i;
     size_t                     j;
     size_t                     k;
+    size_t                     m;
```

Comment:
Hmm, adding variable "m" but not adding any code that uses "m".  Either "m" is a new, unused variable or code existed previously that used an undeclared variable?